scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/Configuration.kt

```diff
@@ -201,7 +209,20 @@ fun KafkaProperties.withStreamThreads(n: Int): KafkaProperties {
     val properties = KafkaProperties()
 
     properties.putAll(this.toMap())
-    this[StreamsConfig.NUM_STREAM_THREADS_CONFIG] = n
```

Comment:
previously, this had no effect (setting the config on "this" does not set it on the returned copy -- the `properties` variable)